clang/lib/CodeGen/CGBuiltin.cpp

```diff
@@ -20723,6 +20723,12 @@ Value *CodeGenFunction::EmitAssignerBuiltinExpr(unsigned int BuiltinID,
     ID = Intrinsic::assigner_exit_check;
     break;
   }
+  case assigner::BI__builtin_assigner_exit_check_eq_pallas: {
+    ID = Intrinsic::assigner_exit_check_eq_pallas;
+    auto ElemTy = llvm::GaloisFieldType::get(context, llvm::GALOIS_FIELD_PALLAS_BASE);
+    OverloadTypes = {ElemTy};
```

Comment:
Any reason why we can't to define it like `BI__builtin_assigner_exit_check` without `OverloadTypes`?
Intrinsic without type specification more convenient: `BI__builtin_assigner_exit_check_eq`